packages/victory-area/src/index.d.ts

```diff
@@ -10,12 +10,14 @@ import {
   VictoryStyleInterface
 } from "victory-core";
 
+export type VictoryAreaTTargetType = "data" | "labels" | "parent";
```

Comment:
Since this is the same for most components, consider moving it to `victory-core`.

Reply:
So my one concern there is that there are a few components for which it's different, and if people want to use the events prop in their own components, they have to import this type specifically for their component. Since this is the only one that's common across many but not universal (like DomainPropType or RangePropType, for example), I just figured it would be easier to find if it stayed in the DefinitelyTyped file for the component? Not married to that idea, though, and will move over if you think that's better.

Reply:
That's a fair point. I'm fine keeping it there.